CHANGES.rst

```diff
@@ -1,11 +1,13 @@
 Changes
 =======
 
-Version 0.7.1 (UNRELEASED)
+Version 0.7.1 (2020-11-09)
 --------------------------
 
 - Adds support for restarting yadage workflows (through ``accept_metadir`` operational option).
 - Allows ``htcondor_max_runtime`` and ``htcondor_accounting_group`` to be specified for HTC jobs.
```

Comment:
Leading "Allows" is not following our "Adds/Fixes/Changes" triad. Should rather be "Adds support for specifying ...".

Also, HTC would beter be spelled HTCondor in full.